chain/parity_wrapper.sh

```diff
@@ -112,12 +98,12 @@ function printHelp() {
 function checkRoleArgument() {
   # Check each known role and end if it match.
   for i in "${VALID_ROLE_LIST[@]}"; do
-    [[ $i == $ROLE ]] && return
+    [[ $i == "$ROLE" ]] && return
   done
 
   # Error report to the user with the correct usage.
   echo "The defined role ('$ROLE') is invalid."
-  echo "Please choose of the following: ${VALID_ROLE_LIST[@]}"
+  echo "Please choose of the following: ${VALID_ROLE_LIST[*]}"
```

Comment:
Please choose **one** of the following **values**

Reply:
:heavy_check_mark: